CONTRIBUTING.md

```diff
@@ -48,7 +48,7 @@ npm test
 ```
 
 ```bash
-# run tests on both React 0.14 and React 0.13
```

Comment:
let's change this to "all supported versions of React" so we don't have to keep editing it as new versions come out :-)